documentation/projects/proposals/document_all_media_properties/20230426-implementation_plan_documenting_media_properties.md

```diff
@@ -61,164 +69,194 @@ presented as a table and will be extracted from the code. Other information
 about the data selection properties and data discrepancies will be written in a
 separate markdown file as it is easier to read, write, and lint.
 
-The catalog has a DAG documentation generator that extracts the docstrings from
-the DAGs and creates a page with the documentation. We can use the same approach
-to create a documentation page for the media properties. The DAG doc generator
-runs the code inside docker to extract DAG information. In this project, we can
-simplify and use the `ast` module to parse static code and extract the
-docstrings to make the checks faster. This will, however, mean that some of the
-settings will not be picked up and will have to be updated during the
-generation. One example of this is the `nullable` field in the `Column` class:
-it falls back to the value of `not nullable` during the object initiation.
-
-## Files
-
-This project will create the following documents:
-
-### `media_properties.md`
+## Output
 
-Located at `catalog/utilities/media_docs_gen/media_properties.md`. This is a
-markdown file with the description of the media properties. This file is
-manually written by the maintainers and contains the detailed information about
-the shape and kind of data we expect to have for the property, how to select it
-from the provider, and any inconsistencies between the data we have in the
-database and the data we expect to have. This file is parsed in step 3 and the
-information is added to the final page on the documentation site.
+The result of this project will be a markdown page with a table of media
+properties and a long-form description of each property.
 
-### `media_properties_documentation.md`
+The page will be published on
+https://docs.openverse.org/openverse_media_properties_documentation.html with
```

Comment:
Based on L82, wouldn't this be?
```suggestion
https://docs.openverse.org/catalog/media_properties_documentation.html with
```

And I think we were discussing putting this under the `/meta`[^1] hierarchy rather than `/catalog`, since we'll have definitions for the other services as well in future IPs that might be useful to have colocated in in the documentation. What do you think?

[^1]: https://github.com/WordPress/openverse/pull/1904#issuecomment-1536588260